pkgs/development/tools/unused/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPackages, cmake }:
+rustPackages.rustPlatform.buildRustPackage rec {
+  pname = "unused";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "unused-code";
+    repo = pname;
+    rev = version;
+    sha256 = "06r6m7k570rdm9szghnp3g4r6ij0vp8apfanqzzxv2hd7gf8v62b";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  cargoSha256 = "1c0gj2wp0nydv0binxj3ikm5sm6y5z3pklp5b06dgvq02licz57a";
+
+  meta = with stdenv.lib; {
+    description = "A tool to identify potentially unused code";
+    homepage = "https://unused.codes";
+    license = licenses.mit;
+    maintainers = [ maintainers.lrworth ];
+    platforms = platforms.all;
```

Comment:
```suggestion
```

Set to the right value by buildRustPackage. It is only required if the set of platforms is smaller than the one set.